packages/authorization/README.md

```diff
@@ -98,8 +98,8 @@ application. The authorization component can be configured with options:
 
 ```ts
 const options: AuthorizationOptions = {
-  precedence: AuthorizationDecisions.DENY;
-  defaultDecision: AuthorizationDecisions.DENY;
+  precedence: AuthorizationDecisions.DENY,
+  defaultDecision: AuthorizationDecisions.DENY
```

Comment:
nitpick: I believe linting rules would add `,` to the end of both lines.  `defaultDecision: AuthorizationDecisions.DENY,`

Reply:
Done thanks